lib/routers/task/index.js

```diff
@@ -55,7 +55,17 @@ router.get('/', (req, res, next) => {
       res.response = response.json.data;
       next();
     })
-    .catch(next);
+    .catch(error => {
+      req.log('error', { ...error, stack: error.stack, message: error.message });
+      res.meta = {
+        filters: [],
+        count: 0,
+        total: 0,
```

Comment:
It kinda sucks that we need to provide all this pointless metadata.

Reply:
yeah... an error was thrown in the pagination if it was missing, and I was too lazy to delve that deep... 😬